pleskdistup/actions/common.py

```diff
@@ -224,3 +226,77 @@ def estimate_post_time(self):
 
     def estimate_revert_time(self):
         return 0
+
+
+class RuleSelinux(action.ActiveAction):
```

Comment:
Hm. it not really makes it permissive, since we return enforce mode after the conversion. I propose `DisableSelinuxDuringPerformance` or something like this.